task/04-arrays-tasks.js

```diff
@@ -353,7 +391,11 @@ function getItemsSum(arr) {
  *  [ null, undefined, NaN, false, 0, '' ]  => 6
  */
 function getFalsyValuesCount(arr) {
-   throw new Error('Not implemented');
+   let s = 0;
+   arr.map(item => {
+      !item ? s++ : null;
+   });
+   return s;
 }
```

Comment:
You can use .filter method to get an array of necessary condition values and then return it with .length,
this note applies to other similar tasks.